src/common/lib/pinata-proxy.js

```diff
@@ -41,7 +41,7 @@ export const uploadFile = val => {
 }
 
 export const getFileUrl = cid => {
-  return `${process.env.VUE_APP_PINATA_GATEWAY}/${cid}`
+  return `${process.env.VUE_APP_PINATA_GATEWAY}/ipfs/${cid}`
```

Comment:
should we use `/ipfs`? I think you need to see this code in the customer repository, I don't think there is a difference there, then you should reference from there

Reply:
if not use `/ipfs` url is ivalid
![image](https://user-images.githubusercontent.com/31072582/165463008-f43a613f-0617-4e1a-a811-95296437db9b.png)


if use `/ipfs` 
![image](https://user-images.githubusercontent.com/31072582/165463143-fe417d13-49d4-4bd3-96b7-9ee423e01888.png)

Reply:
which ENV are you using?

Reply:
oo i see